### DISCUSSION TITLE
Welcome to intensity-normalization Discussions!

### DISCUSSION THREAD

**Discussion by jcreinhold** [jcreinhold/intensity-normalization#45] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community and ask general questions about intensity-normalization. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by esrasmr** [comment]
Hi, 
Thank you for this great intensity-normalization tool. I wonder which sample-based methods can be used directly with images like T2w or FLAIR MR images? As I understood correctly, RAVEL works with pre-contrast T1w but can we apply this tissue membership to T2 images just shown for timepoint-based algorithm FCM? Also, I want to ask your opinion about if we work with multisite MRI data, is it better to fit (e.g. Nyul algorithm) to entire data that coming from the same site, then apply this information to another dataset?
Thank you for your help.

**Reply by jcreinhold** [reply]
Nyul, and perhaps LSQ, are your best bets for sample-based normalization of T2-w and FLAIR images.
